jbi/models.py

```diff
@@ -359,7 +360,25 @@ class ActionContext(Context, extra=Extra.forbid):
 
     rid: str
     operation: Operation
+    current_step: Optional[str]
     event: BugzillaWebhookEvent
     jira: JiraContext
     bug: BugzillaBug
     extra: dict[str, str] = {}
+    responses_by_step: dict[str, list] = {}
+
+    @property
+    def responses(self):
+        """Return flat list of all responses"""
+        # return list(itertools.chain.from_iterable(self.responses_by_step.values()))
```

Comment:
Is this an old implementation and you went for something less complex, i.e. the uncommented code below?